docs/dev-guide/design-overview.md

```diff
@@ -29,48 +31,38 @@ Below, the architecture of the Interoperability Layer is shown in the context of
-
-This component makes use of several other services in order to perform the functions mentioned above. These can be external services and we can likely use existing software components to fulfil these functions. The required services are explained below:
-
-*   Log service - This service stores each message in its entirety along with metadata about the message such as time and date the message was received and the response that the service returned.
-*   Audit service - This service audits each message received by storing an audit log entry. This log entry contains certain key information such as who sent the message, what information was requested and when the information was requested.
-*   Authorization and Authentication service - This service ensures that the person requesting or submitting information is known to the HIE and has the correct privileges to do so.
-
-The interoperability layer core component contacts each one of these services when it receives a message to ensure the appropriate information is stored. It then passes the message on to the router where it is sent to the correct upstream service. The router makes use of a publish and subscribe pattern so that messages can be routed to multiple interested parties. This allows for secondary use of the messages received by the HIE. For example, encounter message could be routed to the SHR as well as an aggregation service where they could be aggregated and submitted to an aggregate data store such as a data warehouse.
+The core provides the key functions and services required for an interoperability layer. These functions are useful in a service-oriented architecture (SOA) environment, which is essentially a collection of services that communicate with each other. 
 
-### The orchestrator and adapter services
+This component can be thought of as the entry point into the HIE. It provides some common mundane services so that other domain services don't have to implement these. This component essentially acts as a web service proxy while performing some additional functions on the incoming requests. It makes use of several other services in order to perform the functions mentioned below: 
 
-This set of components provides services that manipulate the requests that are sent to them. They are often implementation specific so they will change as the use cases that the HIE supports change. Each of these components are separate, independent services that perform a specific function following the micro services architecture ([click here for additional information about mico service architectures](http://yobriefca.se/blog/2013/04/29/micro-service-architecture/)). There are 2 major types of these services:
+* Authorisation and Authentication - This service ensures that the client (human user or system) requesting or submitting information is known to the HIE is known to it and has the correct privileges or permissions to do so.
```

Comment:
Im not sure if this sentence reads correctly at this point? 
`... is known to the HIE is known to it ...`